database/seeders/DatabaseSeeder.php

```
@@ -15,5 +17,15 @@
     public function run(): void
     {
         Contact::factory(10)->create();
+
+        $countries = countries();
+        foreach ($countries as $country) {
+            dump($country);
+            $c = new CountryModel;
+            $c->name = $country['name'];
+            $c->iso_code = $country['iso_3166_1_alpha2'];
+            $c->data = $egypt = country($country['iso_3166_1_alpha2'])->getAttributes();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$egypt'. (Unused Code Rules)

(UnusedLocalVariable)